refactor remote access guide (#453)
zbm-dev/zfsbootmenu: docs/guides/general/remote-access.rst

      lines (five, if you count the harmless comment) of the ``install()`` functioned. Removing these lines should not be
      critical, as Dracut should happily continue the initramfs creation process even if those installation commands fail.

    If you use Dracut to produce the initramfs images in your boot environment, you may wish to disable the ``crypt-ssh``
    module in those images. Just add::

      omit_dracutmodules+=" crypt-ssh "

    to a configuration file in ``/etc/dracut.conf.d``. The configuration file must have a ``.conf`` extension to be
    recognized; see :manpage:`dracut.conf(5)` for more information.

    To inform ``dracut-network`` that it must bring up a network interface, pass the kernel command-line parameters
    ``ip=dhcp`` and ``rd.neednet=1`` to your ZFSBootMenu image. If you use another boot loader to start ZFSBootMenu, *e.g.*
    rEFInd or syslinux, this can be accomplished by configuring that loader. However, it may be more convenient to add these
    parameters directly to the ZFSBootMenu image::

      mkdir -p /etc/cmdline.d

>       echo "ip=dhcp rd.neednet=1" > /etc/cmdline.d/dracut-network.conf

Reviewer: Won't this add this to both the ZBM image and to the distro/BE image?

Reviewer 2: I think these files need to be explicitly added to images, so they should be OK here.

    It is possible to specify a static IP configuration by replacing ``dhcp`` with a properly formatted configuration
    string. Consult the `dracut documentation <https://man.voidlinux.org/dracut.cmdline.7#Network>`_ for details about
    static IP configuration.

    There are methods besides writing to ``/etc/cmdline.d`` or configuring another boot loader to specify kernel
    command-line arguments that will configure networking in Dracut. However, Dracut uses the ``/etc/cmdline.d`` directory

>     to store "fake" arguments, which it processes directly rather than handing to the kernel. In my tests, using other

Reviewer 2: ```suggestion
    to store "fake" arguments, which it processes directly rather than handing to the kernel. Using other
```

          <https://wiki.archlinux.org/title/Mkinitcpio#Using_net>`_.

      .. group-tab:: mkinitcpio-rclocal

        First, install ``mkinitcpio-rclocal``::

          curl -L https://github.com/ahesford/mkinitcpio-rclocal/archive/master.tar.gz | tar -zxvf - -C /tmp
          mkdir -p /etc/zfsbootmenu/initcpio/{install,hooks}
          cp /tmp/mkinitcpio-rclocal-master/rclocal_hook /etc/zfsbootmenu/initcpio/hooks/rclocal
          cp /tmp/mkinitcpio-rclocal-master/rclocal_install /etc/zfsbootmenu/initcpio/install/rclocal
          rm -r /tmp/mkinitcpio-rclocal-master

        Next, create an ``rc.local`` script that can be run within the mkinitcpio image to configure the ``eth0`` interface::

          cat > /etc/zfsbootmenu/initcpio/rc.local <<RCEOF
          #!/bin/sh

          # Don't attempt to configure an interface that does not exist
          ip link show dev eth0 >/dev/null 2>&1 || exit

          # Bring up the interface
          ip link set dev eth0 up

          # Configure a static address for this host
          ip addr add 192.168.1.2/24 brd + dev eth0
          ip route add default via 192.168.1.1

          # Add some name servers

>           cat > /etc/resolv.conf <<-EOF
>             nameserver 1.1.1.1
>             nameserver 8.8.8.8
>             EOF

Reviewer 2: ```suggestion
          cat > /etc/resolv.conf <<-EOF
          	nameserver 1.1.1.1
          	nameserver 8.8.8.8
          	EOF
```
Not sure if Sphinx will preserve a tab inside this block, but I think the shell will refuse to recognize the indented heredoc unless real tabs are used. That could bite people copying and pasting this block if we can't include tabs.

Author: it's not preserved, I think we can just use a `<<` heredoc instead

    inside a mkinitcpio image. This package is based on a `project of the same name
    <https://github.com/grazzolini/mkinitcpio-dropbear>`_ by an Arch Linux developer. A `fork of the mkinitcpio-dropbear
    project <https://github.com/ahesford/mkinitcpio-dropbear>`_ contains a few minor improvements in runtime
    configuration and key management. If these improvements are not needed, using the upstream project is perfectly
    acceptable.

    First, download and install the mkinitcpio module::

      curl -L https://github.com/ahesford/mkinitcpio-dropbear/archive/master.tar.gz | tar -zxvf - -C /tmp
      mkdir -p /etc/zfsbootmenu/initcpio/{install,hooks}
      cp /tmp/mkinitcpio-dropbear-master/dropbear_hook /etc/zfsbootmenu/initcpio/hooks/dropbear
      cp /tmp/mkinitcpio-dropbear-master/dropbear_install /etc/zfsbootmenu/initcpio/install/dropbear
      rm -r /tmp/mkinitcpio-dropbear-master

    Then, enable the ``dropbear`` module in ``/etc/zfsbootmenu/mkinitcpio.conf`` by manually appending ``dropbear`` to
    the ``HOOKS`` array, or by running::

      sed -e '/HOOKS.*rclocal/a HOOKS+=(dropbear)' -i /etc/zfsbootmenu/mkinitcpio.conf

    .. note::

>       The default configuration will start dropbear on TCP port 22. If using the ``ahesford/mkinitcpio-dropbear`` for

Reviewer 2: ```suggestion
      The default configuration will start dropbear on TCP port 22. If using the ``ahesford/mkinitcpio-dropbear`` fork
```

>       The default configuration will start dropbear on TCP port 22. If using ahesford's fork of ``mkinitcpio-dropbear``,

Reviewer 2: ```suggestion
      The default configuration will start dropbear on TCP port 22. If using the ``ahesford/mkinitcpio-dropbear`` fork recommended here,
```
